### DISCUSSION TITLE
sin.x and sin.y

### DISCUSSION THREAD

**Discussion by Finally1994** [ikemen-engine/Ikemen-GO#1160] [Feature Request]
### Describe the solution you'd like

sin.x and sin.y used in stage.def etc., can be used in character state controllers.
I want to make a meandering helper and a projectile in a simple way.

**Comment by leanycat** [comment]
there's already a way to use sin in parameters: sin([condition goes here]). same as cos and tan, it's documented in the mugen docs

example:
VelSet{x: sin(time/8)}

**Reply by Finally1994** [reply]
Thanks for letting me know
So it will be helpful
